packages/sanity/src/core/store/bundles/__workshop__/BundlesStoreStory.tsx

```diff
@@ -90,19 +107,57 @@ const BundlesStoreStory = () => {
             <Card key={bundle._id} padding={3} border radius={3}>
               <Flex align="center" gap={3} justify={'space-between'}>
                 <Text>{bundle.name}</Text>
-                <Button
-                  text="Delete"
-                  tone="critical"
-                  // eslint-disable-next-line react/jsx-no-bind
-                  onClick={() => handleDeleteBundle(bundle._id)}
-                  disabled={deleting === bundle._id}
-                  loading={deleting === bundle._id}
-                />
+                <Flex align="center" gap={2}>
+                  <Button
+                    text="Delete"
+                    mode="bleed"
+                    tone="critical"
+                    // eslint-disable-next-line react/jsx-no-bind
+                    onClick={() => handleDeleteBundle(bundle._id)}
+                    disabled={deleting === bundle._id}
+                    loading={deleting === bundle._id}
+                  />
+                  <Button
+                    text="Edit"
+                    // eslint-disable-next-line react/jsx-no-bind
+                    onClick={() => setEditing(bundle)}
+                    disabled={deleting === bundle._id}
+                    loading={deleting === bundle._id}
+                  />
+                </Flex>
               </Flex>
             </Card>
           ))}
         </Stack>
       </Card>
+      {editing && (
+        <Dialog
+          id="edit-bundle-dialog"
+          header="Edit release"
+          onClose={() => setEditing(null)}
+          footer={{
+            confirmButton: {
+              onClick: () => handleUpdateBundle(editing),
+              disabled: updating,
+              loading: updating,
+              tone: 'primary',
+            },
+          }}
+        >
+          {editing && (
+            <BundleForm
+              // eslint-disable-next-line react/jsx-no-bind
+              onChange={(bundle) =>
+                setEditing((prev) => ({
+                  ...(prev as BundleDocument),
+                  ...bundle,
+                }))
+              }
+              value={editing}
+            />
+          )}
+        </Dialog>
+      )}
```

Comment:
This will allow us to edit the bundles from the Story, it would be nice to later extend this to include more fields and make it easier to test the UI with real data